services/go-importer/cmd/assembler/main.go

```diff
@@ -208,6 +283,16 @@ func handlePcap(fname string) {
 		bytes += int64(len(data))
 		done := false
 
+		if !nextFlush.IsZero() {
+			// Check to see if we should flush the streams we have that haven't seen any new data in a while.
+			// Note that pcapOpenOfflineFile is blocking so we need at least see some packets passing by to get here.
+			if time.Now().After(nextFlush) {
+				log.Printf("flushing all streams that haven't seen packets in the last %s", *flushAfter)
+				assembler.FlushCloseOlderThan(time.Now().Add(flushDuration))
```

Comment:
Make sure to flush *older* than not into the future :)
```suggestion
				assembler.FlushCloseOlderThan(time.Now().Add(-flushDuration))
```